go.mod

```diff
@@ -79,3 +79,7 @@ require (
 replace github.com/hashicorp/go-immutable-radix => github.com/tonistiigi/go-immutable-radix v0.0.0-20170803185627-826af9ccf0fe
 
 replace github.com/jaguilar/vt100 => github.com/tonistiigi/vt100 v0.0.0-20190402012908-ad4c4a574305
+
+replace github.com/containerd/containerd => github.com/containerd/containerd v1.3.1-0.20191014053712-acdcf13d5eaf
```

Comment:
`github.com/containerd/containerd v1.4.0-0.20191014053712-acdcf13d5eaf` in L13 should be like `github.com/containerd/containerd v0.0.0`?

Reply:
Is there a case that requires this. As long as replace has the "sanitized" version it seems to work for me.

Reply:
Just for human understandability

Reply:
`v1.4.0-0` might improve understandability as well, as it is "newer than 1.3"